sorc/ncep_post.fd/CLDRAD.f

```diff
@@ -2047,15 +2004,26 @@ SUBROUTINE CLDRAD
                 GRID1(I,J) = CLDZ(I,J)
               ENDDO
             ENDDO
-               if(grib=="grib1" )then
-               ID(1:25)=0
-               CALL GRIBIT(IGET(408),LVLS(1,IGET(408)),GRID1,IM,JM)
-               else if(grib=="grib2" )then
+               if(grib=="grib2" )then
                  cfld=cfld+1
                  fld_info(cfld)%ifld=IAVBLFLD(IGET(408))
                  datapd(1:im,1:jend-jsta+1,cfld)=GRID1(1:im,jsta:jend)
                endif
           ENDIF
+!   GSD CLOUD BOTTOM PRESSURE
+          IF (IGET(798).GT.0) THEN
+!$omp parallel do private(i,j)
+            DO J=JSTA,JEND
+              DO I=1,IM
+                GRID1(I,J) = CLDP(I,J)
+              ENDDO
+            ENDDO
+               if(grib=="grib2" )then
+                 cfld=cfld+1
+                 fld_info(cfld)%ifld=IAVBLFLD(IGET(798))
+                 datapd(1:im,1:jend-jsta+1,cfld)=GRID1(1:im,jsta:jend)
+               endif
+          ENDIF
```

Comment:
@JesseMeng-NOAA It seems to me you haven't synced the latest changes in CLDRAD.f from upstream develop.

Reply:
Updated. Thanks!